packages/next-swc/crates/next-api/src/app.rs

```diff
@@ -894,8 +896,31 @@ impl AppEndpoint {
             ));
             server_assets.push(app_build_manifest_output);
 
+            const POLYFILL_PATH: &str = "next/dist/build/polyfills/polyfill-nomodule.js";
+            let project_path = this.app_project.project().project_path();
+            let polyfill_path = resolve(
```

Comment:
You don't really need to use resolving at all. You could use `get_next_package()` to get the next.js package path and `join` the remaining path.

Then convert it via `FileSource` and `RawOutputAssetA`

Reply:
I did see `get_next_package()`, but hesitated to use it because while it is public within the `next_core` crate, the `next_import_map` module itself isn't public.

I do think it makes sense to only resolve the next package once though, so I'll re-export it.